### PR TITLE
Calibrated For JSON data

### PR DESCRIPTION
**Pull request by abhijeet-chatterjee**

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -66,30 +68,30 @@ def processurl(urltsm,agent):
...
-                if RESULT['scans'][i]['detected'] == True:
-                    TRIGGERS.append(i)
-                    MALHITS.append(RESULT['scans'][i]['result'])
-    else:
-        TRIGGERS=[]
-        MALHITS=[]
-        TRIGGERS.append("--")
-        MALHITS.append("--")
-        REASON['malware']="VirusTotal API key not used, skipping malware scan."
+    #    if MALWARESCORE > 0:
```
Comment (Saket-Upadhyay) [inline app.py]: Why you commented out Malware scanning part?

File: app.py [hunk 2]
```diff
@@ -225,13 +227,15 @@ def processurl(urltsm,agent):
...
-                           HTTPSSCORE=HTTPSSCORE,COVRED=COVRED,REDIR=REDIR,CERT0=CERT0,CERT1 = CERT1,CERT2 = CERT2,CERT3 = CERT3,CERT4 = CERT4,CERT5 = CERT5,CERT6 = CERT6,CERTTRUST=CERTTRUST
-                           ,ERROR404=ERROR404,REASON=REASON)
-
+    #if agent == "shell":
+    #   return str(str(TRANRESULT)+";"+(WEBUS)+";"+(MALSR)+";"+str(TRIGGERS)+";"+str(MALHITS)+";"+HTTPSSCORE+";"+COVRED+";"+REDIR+";"+CERT0+";"+CERT1+";"+CERT2+";"+CERT3+";"+CERT4+";"+CERT5+";"+CERT6+";"+CERTTRUST+";"+str(ERROR404)+";"+str(REASON))
+
+    #return render_template('res.html',TRANRESULT=TRANRESULT,WEBUS=WEBUS,MALSR=MALSR,TRIGGERS=TRIGGERS,MALHITS=MALHITS,
+     #                      HTTPSSCORE=HTTPSSCORE,COVRED=COVRED,REDIR=REDIR,CERT0=CERT0,CERT1 = CERT1,CERT2 = CERT2,CERT3 = CERT3,CERT4 = CERT4,CERT5 = CERT5,CERT6 = CERT6,CERTTRUST=CERTTRUST
+      #                    ,ERROR404=ERROR404,REASON=REASON)
+    data={"TRANRESULT":TRANRESULT,"WEBUS":WEBUS,"HTTPSSCORE":HTTPSSCORE,"COVERT":COVRED,"REDIR":REDIR,"CERT0":CERT0,"CERT1":CERT1,"CERT2":CERT2,
```
Comment (Saket-Upadhyay) [inline app.py:236]: MALSR is important result. please check why it's not in response

Comment (Saket-Upadhyay) [reply app.py:236]: TRIGGERS, MALHITS and REDIR parameters missing from json response

**Review by Saket-Upadhyay — CHANGES_REQUESTED** [review]
Check the marked review comments, there are some basic issues i think. rest is ok.

File: app.py (hunk 2 above)
Comment (abhijeet-chatterjee) [reply app.py:236]: My bad didn't notice you fixes the virustotal api part making another

**Pull request closed without merging**